src/components/publiq-ui/Modal/QuestionModal.js

```diff
@@ -83,9 +84,7 @@ QuestionModal.defaultProps = {
   title: '',
   confirmTitle: 'Ok',
   cancelTitle: 'Cancel',
-  onShow: () => {},
-  onClose: () => {},
-  onConfirm: () => {},
+  size: 'sm',
```

Comment:
Same question if we can use `ModalSizes.SM` here

Reply:
Because the ModalSize lives in Another component `Modal`, It would throw error's that the constant was used before it was defined